Automattic/wp-calypso
# Store Orders: Extract ActionHeader on single order views to its own component

This cleans up the parent order component somewhat by moving the ActionHeader, buttons, and related actions (edit, save, cancel, send invoice) to a new component. It also removes the config check for `woocommerce/extension-orders-edit`, since that's been live to production for a while now :) There should be no other functionality change.

![screen shot 2018-02-01 at 7 16 25 pm](https://user-images.githubusercontent.com/541093/35710246-6da499ca-0784-11e8-9f87-a0f36fe23199.png)

**To test:**

- View a single order: Go to orders `/store/orders/:siteId` & click into one
- The header should look like above, with breadcrumb nav & and edit button
- If the order is pending payment, you should also see a button to send an invoice
- Make sure the edit button toggles edit state
- Now you should see a "Save" & "Cancel"
- Check that these work
- Go back to a pending payment order & check that the Send Invoice button works
